### PR TITLE
Ui/improve schedule

### PR DESCRIPTION
**Pull request by hellais**

### CODE REVIEW

File: proteus-frontend/components/meta.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import NProgress from 'nprogress'
+import Router from 'next/router'
+
+Router.onRouteChangeStart = () => NProgress.start()
+Router.onRouteChangeComplete = () => NProgress.done()
+Router.onRouteChangeError = () => NProgress.done()
+
+export default () => (
+  <div>
+		<Head>
```
Comment (bassosimone) [inline proteus-frontend/components/meta.js]: Shouldn't this be `<head>`?

**Comment by bassosimone** [comment]
@hellais can you merge master into this PR?

File: proteus-frontend/components/meta.js (hunk 1 above)
Comment (hellais) [inline proteus-frontend/components/meta.js]: No, `<head>` is the HTML tag, `<Head>` is the special `next.js` component that is used to inject stuff into the `<head>`.

Comment (bassosimone) [inline proteus-frontend/components/meta.js]: Oh, this is not confusing at all :-D

[ep052](http://geekz.co.uk/lovesraymond/archive/mundane-name)

Comment (hellais) [inline proteus-frontend/components/meta.js]: Well it effectively works the same way as `<head>`, except you can call it multiple times. In react-land it's a well established convention to call component names starting with an upper-case character, so it's not that odd.

Comment (bassosimone) [inline proteus-frontend/components/meta.js]: Yeah, it's similar to `String string`.

**Pull request merged**